benchmark/scripts/Benchmark_Driver

```diff
@@ -227,24 +236,28 @@ def submit(args):
     submit_to_lnt(data, args.lnt_host)
     return 0
 
+
 def run(args):
     optset = args.optimization
     file = os.path.join(args.tests, "Benchmark_" + optset)
     run_benchmarks(file, benchmarks=args.benchmarks,
-            num_samples=args.iterations, verbose=True,
-            log_directory=args.output_dir,
-            swift_repo=args.swift_repo)
+                   num_samples=args.iterations, verbose=True,
+                   log_directory=args.output_dir,
+                   swift_repo=args.swift_repo)
```

Comment:
Alternatively, this could be formatted like so:

```py
run_benchmarks(
    file, benchmarks=args.benchmarks,
    num_samples=args.iterations,
    verbose=True,
    log_directory=args.output_dir,
    swift_repo=args.swift_repo)
```

I think the rule simply requires that each line of parameters begin at the same indentation. The "ideal" indentation here is subjective, but I think this indentation rule and the 80 columns rule can co-exist.